systemtest/src/test/java/io/strimzi/systemtest/operators/RecoveryST.java

```diff
@@ -71,7 +66,6 @@ void testRecoveryFromKafkaStrimziPodSetDeletion() {
     }
 
     @IsolatedTest("We need for each test case its own Cluster Operator")
-    @KRaftNotSupported("Zookeeper is not supported by KRaft mode and is used in this test class")
```

Comment:
**regarding allowing tests in kraft:** 
- tests with services do not make much sense in kraft as these services are no longer present. 
- test of pod sets would make sense but it would still fail as currently we target specifically zookeeper so this needs adjustment to target rather controller by selector.  e.g. to target strimzi pod set for controller `my-cluster-7-c-9aa820e3` instead of zookeeper it expects something like `my-cluster-7-zookeeper`.